roles/node_images_metal/files/scripts/metal/create-iso-artifact.sh

```diff
@@ -33,6 +33,9 @@ ARCH="${ARCH:-}"
 if [ -z "${ARCH}" ]; then
     ARCH="$(uname -m)"
 fi
+if [ "${ARCH}" = "aarch64" ]; then
```

Comment:
```suggestion
elif [ "${ARCH}" = "aarch64" ]; then
```

Reply:
That isn't the logic I'm looking for. We need to set ARCH if it's unset, then if ARCH is aarch64, change it to arm64.